great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -83,8 +92,6 @@ def get_line_chart(
         metric_type: alt.StandardType,
         domain_name: str,
         domain_type: alt.StandardType,
-        line_color: Optional[str] = Colors.BLUE_2.value,
-        point_color: Optional[str] = Colors.GREEN.value,
         point_color_condition: Optional[alt.condition] = None,
```

Comment:
I'd like point_color_condition and tooltip to be added to the theme rather than being here; having Altair-specific parameters leaks implementation and adds to the burden of the caller. Our default theme should include these values